pkg/kubelet/images/image_manager.go

```diff
@@ -174,6 +168,23 @@ func (m *imageManager) EnsureImageExists(ctx context.Context, pod *v1.Pod, conta
 	return imagePullResult.imageRef, "", nil
 }
 
+func evalCRIPullErr(container *v1.Container, err error) (errMsg string, errRes error) {
+	// Error assertions via errors.Is is not supported by gRPC (remote runtime) errors right now.
+	// See https://github.com/grpc/grpc-go/issues/3616
+	if err.Error() == ErrRegistryUnavailable.Error() {
```

Comment:
should those errors be defined in CRI API package or have a comment for API list for OOMKilled?

https://github.com/kubernetes/kubernetes/blob/a6825c8542ee1e5e9d570ba1c267458a09287015/staging/src/k8s.io/cri-api/pkg/apis/runtime/v1/api.proto#L1230

Reply:
I moved them into the CRI API `errors` package, would that fit better?